src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -149,9 +155,13 @@ def save_table(self, full_name: str, rows: list[any], mode: str = "append"):
         df = self._spark.createDataFrame(rows, self._schema_for(rows[0]))
         df.write.saveAsTable(full_name, mode=mode)
 
+    def create_empty_table(self, full_name: str, klass):
+        ddl = f"CREATE TABLE IF NOT EXISTS {full_name} ({self._schema_for(klass)}) USING DELTA"
+        self.execute(ddl)
+
 
 class CrawlerBase:
-    def __init__(self, backend: SqlBackend, catalog: str, schema: str, table: str):
+    def __init__(self, backend: SqlBackend, catalog: str, schema: str, table: str, klass=None):
```

Comment:
```suggestion
    def __init__(self, backend: SqlBackend, catalog: str, schema: str, table: str, klass):
```
Why klass is not required? It's needed everywhere, right?..

Reply:
I did it for legacy support. I should really not be lazy and change it everywhere...